jira_scraper/processors/vector_db.py

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
We don't need a class for this and neither the module.

Reply:
This separates qdrant api specific code int it's own class. It would make it easier to switch and extensible to use any other vectordb or use different databases for different things in the future.